tox.ini

```diff
@@ -8,17 +8,15 @@ norecursedirs = .* docs requirements
 
 [testenv]
 deps = 
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
-	-r{toxinidir}/requirements/test.txt
+    django32: Django>=3.2,<4.0
+    django40: Django>=3.3,<4.1
+    -r{toxinidir}/requirements/test.txt
 commands = 
-	pytest {posargs}
+    pytest {posargs}
 
 [testenv:quality]
 commands = 
-	pycodestyle --config=.pep8 edx_rest_api_client
-	pylint --rcfile=pylintrc edx_rest_api_client
-	twine check {distdir}/*
```

Comment:
we can let these changes be if we can verify that these indentation changes are a one time change and if we run the modernizers again modernizers won't make this change

Reply:
These are just one time changes and won't happen again. I have verified this